website/static/css/pages/share-buttons.css

```diff
@@ -0,0 +1,41 @@
+div.dropdown .dropdown-menu li, .shareButtons {
+    margin: 3px 20px;
+}
+
+div.dropdown .dropdown-menu li a, .shareButtons a {
```

Comment:
Is it necessary to select `div.dropdown...`? I'm concerned that overriding Bootstrap's defaults might have unwanted side effects.